src/context.ts

```diff
@@ -126,6 +126,7 @@ class Context extends EventEmitter {
    */
   stash: Record<string, any> = {};
 
+  _cachedFunctions: Record<string, Record<string, MojoAction>> = {};
```

Comment:
These private properties should probably reflect that they are for nested helpers specifically.

Reply:
Like `_nestedHelperCache` or so.